### PR TITLE
adding heart disease prediction

### LINKED ISSUE #618: Heart disease prediction
# Aim

- Heart disease prediction using machine learning

# Details

- This script predicts whether a person has heart disease or not using machine learning

### PR DESCRIPTION
**Pull request by aaadddiii**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Heart Disease Prediction/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Heart disease prediction
```
Comment (kaustubhgupta) [inline Heart Disease Prediction/README.md:1]: After this heading, all the headings should be at level two (##)

File: Heart Disease Prediction/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,80 @@
+asgiref @ file:///tmp/build/80754af9/asgiref_1594338739818/work
```
Comment (kaustubhgupta) [inline Heart Disease Prediction/requirements.txt:1]: The requirements file is overpopulated with unnecessary modules. Clear it and make sure to include only those modules which are used in the script

File: Heart Disease Prediction/script.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+data["oldpeak"] = float(input())
+print(
+    "[+] Enter the slope of the peak exercise ST segment \n Value 1: upsloping \n Value 2: flat \n Value 3: downsloping"
+)
+data["slope"] = int(input())
+print("[+] Enter number of major vessels (0-3) colored by flourosopy")
+data["ca"] = int(input())
+print("[+] Enter 3 = normal, 6 = fixed defect, 7 = reversable defect")
+data["thal"] = int(input())
+model = pickle.load(open("./heart_model.pkl", "rb"))
```
Comment (kaustubhgupta) [inline Heart Disease Prediction/script.py:39]:
```suggestion
model = pickle.load(open("./Heart Disease Prediction/heart_model.pkl", "rb"))
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Minor changes needed:

File: Heart Disease Prediction/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+```
+Install the requirements 
+```bash
+pip install -r requirements.txt
+```
+## To run
+```bash
+python script.py
+```
+#### By [Adithya Kurien Peter]("https://github.com/aaadddiii")
```
Comment (kaustubhgupta) [inline Heart Disease Prediction/README.md:23]: Level 2 (##)

File: Heart Disease Prediction/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+numpy==1.18.5
+packaging==20.4
```
Comment (kaustubhgupta) [inline Heart Disease Prediction/requirements.txt:2]: not needed

File: Heart Disease Prediction/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+numpy==1.18.5
```
Comment (kaustubhgupta) [inline Heart Disease Prediction/requirements.txt:1]: Replace all these with this list:
numpy
pandas
scikit-learn

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Heart Disease Prediction/README.md (hunk 4 above)
Comment (aaadddiii) [reply Heart Disease Prediction/README.md]: Iam sorry , Have made the changes

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**